engines/ic3base.cpp

```diff
@@ -129,10 +129,11 @@ ProverResult IC3Base::check_until(int k)
   int i = reached_k_ + 1;
   assert(i >= 0);
   while (i <= k) {
-    // reset cex_pg_ to null
+    // reset cex_pg_->to null
     // there might be multiple abstract traces if there's a derived class
     // doing abstraction refinement
-    cex_pg_ = ProofGoal();
+    cex_pg_ = new ProofGoal();
```

Comment:
where do we  delete ?

Reply:
Oops, good catch. Thanks! Fixed with 5c927cdc9fbcad0b3f270f0ba117cd447278a72a.